src/Agent/NewRelic/Agent/Parsing/MySqlExplainPlanActions.cs

```diff
@@ -51,9 +51,13 @@ public static ExplainPlan GenerateExplainPlan(object resources)
             using (dbCommand)
             using (dbCommand.Connection)
```

Comment:
Not related to your changes, but should the 'dbcommand.connection.Open()' on line 46 be INSIDE this using block?

Reply:
You know that does look a little odd.... let me read through it some more...

Reply:
Great catch Josh! I moved the connection.open calls into the using, after the parameter check.  If the connection is closed and we don't run a plan, we will now save a DB connection.